workflow/kfdrc_RNAseq_workflow.cwl

```diff
@@ -564,13 +586,24 @@ outputs:
       \ 10 or more read counts of support"}
 
 steps:
+  basename_picker:
+    run: ../tools/basename_picker.cwl
+    in:
+      read1_filename:
+        source: reads1
+        valueFrom: $(self.basename.split('.')[0])
```

Comment:
The input parameter name "read1_filename" would imply the full filename including extension. I would change this to "read1_basename" or, ideally, pass the full filename to basename_picker and pull out the basename within that task.

Reply:
To be accurate, basename also implies full filename including extensions according to CWL spec. I've named it root_name for now.

Reply:
Oh yeah, that is true. root_name is the right way to go. Approving